feathr_project/feathr/_synapse_submission.py

```diff
@@ -336,7 +337,7 @@ def upload_file(self, src_file_path)-> str:
 
     def download_file(self, target_adls_directory: str, local_dir_cache: str):
         """
-        Download file to a local cache
+        Download file to a local cache. Supporting download a folder and the content in its subfolder
```

Comment:
adding all its subfolders recursively (to be more precise)

Reply:
Actually no. The code will just download the content in the root folder, and the folder in the next level (rather than recursively for all layers of folders)

Reply:
I've updated the code comments to make it more clear